tools/database/src/corrupt.rs

```diff
@@ -0,0 +1,39 @@
+use crate::utils::open_state_snapshot;
+use clap::Parser;
+use near_primitives::shard_layout::ShardLayout;
+use near_store::{flat::FlatStorageManager, ShardUId, StoreUpdate};
+use std::path::PathBuf;
+
+#[derive(Parser)]
+pub(crate) struct CorruptStateSnapshotCommand {}
+
+impl CorruptStateSnapshotCommand {
+    pub(crate) fn run(&self, home: &PathBuf) -> anyhow::Result<()> {
+        let store = open_state_snapshot(home, near_store::Mode::ReadWrite)?;
+        let flat_storage_manager = FlatStorageManager::new(store.clone());
+
+        let mut store_update = store.store_update();
+        // TODO(resharding) there must be a better way to get the shard uids
```

Comment:
This highly depends on what shard version we are trying to corrupt right? Could we pass in the shard_version as input here maybe and use that to get the shard_uids?

Reply:
Not a bad workaround at all, though I prefer to keep the cmdline as user friendly as possible. 
It's not a top priority right now so I'll leave that in and we can clean up after all else is done.